src/alpaka/plugin-SiPixelClusterizer/gpuClustering.h

```diff
@@ -0,0 +1,353 @@
+#ifndef RecoLocalTracker_SiPixelClusterizer_plugins_gpuClustering_h
+#define RecoLocalTracker_SiPixelClusterizer_plugins_gpuClustering_h
+
+#include <cstdint>
+#include <cstdio>
+
+#include "AlpakaCore/alpakaConfig.h"
+#include "AlpakaCore/alpakaWorkDivHelper.h"
+#include "AlpakaCore/HistoContainer.h"
+#include "AlpakaDataFormats/gpuClusteringConstants.h"
+#include "Geometry/phase1PixelTopology.h"
+
+namespace gpuClustering {
+
+#ifdef GPU_DEBUG
+  ALPAKA_STATIC_ACC_MEM_GLOBAL uint32_t gMaxHit = 0;
```

Comment:
It is defined here: https://github.com/ghugo83/alpaka/blob/develop/include/alpaka/core/Common.hpp#L88

Reply:
Thanks. I think it should be moved into `ALPAKA_ACCELERATOR_NAMESPACE` because otherwise same symbol would be defined e.g. for serial and TBB backends (leading to ODR violation) when we (later) try to link all of them together. But as I said before, not important for now because if being needed only for debugging, but perhaps a comment here would help to "remember" later?

Reply:
Yes makes sense, ok will add that